activity_browser/bwutils/superstructure/file_imports.py

```diff
@@ -142,7 +143,7 @@ def database_and_key_check(data: pd.DataFrame) -> None:
                     msg = "Error in importing file with activity {} and {}".format(ds[4], ds[5])
                     raise IncompatibleDatabaseNamingError()
         except IncompatibleDatabaseNamingError as e:
-            print(msg)
+            log.error(msg)
```

Comment:
Part of the reason is that this error doesn't actually contain any information. The decision I made previously was to provide a statement of what caused the error when and where it happened and to raise an error that could be captured at a higher level for managing what should happen. This will be redundant when I push up the changes for scenario files soon, for the sake of doing things properly this should probably take the Exception, however for the sake of expediency I'm going to avoid this for now